.ci/publish-aws.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+#
+# Publishes the created lambda layer zip to AWS as AWS Lambda Layers in every region.
+# Finalized by generating an ARN table which will be used in the release notes.
+#
+# AWS_FOLDER is used for temporary output of publishing layers used to create the arn table. (Optional)
+# ELASTIC_LAYER_NAME is the name of the lambda layer e.g. elastic-apm-java-ver-3-44-1 for the git tag v3.44.1 (Required)
+
+
+# This needs to be set in GH actions
+# https://dotjoeblog.wordpress.com/2021/03/14/github-actions-aws-error-exit-code-255/
+# eu-west-1 is just a random region
+export AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION:-eu-west-1}
+
+export AWS_FOLDER=${AWS_FOLDER:-.ci/.aws}
+FULL_LAYER_NAME=${ELASTIC_LAYER_NAME:?layer name not provided}
+ALL_AWS_REGIONS=$(aws ec2 describe-regions --output json --no-cli-pager | jq -r '.Regions[].RegionName')
+
+rm -rf "${AWS_FOLDER}"
+mkdir -p "${AWS_FOLDER}"
+
+# Delete previous layers
```

Comment:
Isn't automatically deleting stuff a bit dangerous?

Reply:
this was a copy-paste from another project.

But we can remove it.

https://github.com/elastic/apm-agent-java/pull/3185/commits/335305adc652d20259c2c8cab2f492393b7b1611